vehicles/vehiclestate.h

```diff
@@ -73,6 +73,9 @@ class VehicleState : public ObjectState
     void setIsArmed(bool isArmed);
     void setAutopilotRadius(double radius);
     double getAutopilotRadius();
+    bool getHasTrailer() const;
+    void setHasTrailer(bool mHasTrailer);
+    virtual double getTrailerAngleRadians() const {return 0.0;}
```

Comment:
I think its a good idea to add a private boolean variable mHasTrailer with public get & set methods, but it should be in truckstate, given its vehicle specific